### ISSUE TITLE
Item numbers does not report correctly on IAccessible2

### ISSUE THREAD

**Issue by OzancanKaratas** [nvaccess/nvda#11117]
### Steps to reproduce:
Start regedit.exe, and navigate the tree view.
### Actual behavior:
NVDA announces as 0 items everything.
### Expected behavior:
NVDA should be announced correctly the item numbers.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2019.3.1
#### Windows version:
Windows 10 version 1909
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Same in the newer versions as well.
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes

**Comment by zahra21** [comment]
hi.
do you use 64 bit of windows?
i read in some similar issues that this problem occures in 64 bit windows, because nvda is 32 bit application!

**Comment by Adriani90** [comment]
Closing as duplicate of #7041.
